src/connector/src/sink/iceberg/mod.rs

```diff
@@ -1273,29 +1305,45 @@ impl SinkCommitCoordinator for IcebergSinkCommitter {
             })
             .collect::<Result<Vec<DataFile>>>()?;
 
-        let txn = Transaction::new(&self.table);
-        let mut append_action = txn
-            .fast_append(None, vec![])
-            .map_err(|err| SinkError::Iceberg(anyhow!(err)))?;
-        append_action
-            .add_data_files(data_files)
-            .map_err(|err| SinkError::Iceberg(anyhow!(err)))?;
-        let tx = append_action.apply().await.map_err(|err| {
-            tracing::error!(error = %err.as_report(), "Failed to commit iceberg table");
-            SinkError::Iceberg(anyhow!(err))
-        })?;
-        let table = tx
-            .commit_dyn(self.catalog.as_ref())
-            .await
-            .map_err(|err| SinkError::Iceberg(anyhow!(err)))?;
-        self.table = table;
+        let mut success_commit = false;
+        for t in 0..self.commit_retry_num {
```

Comment:
Should we introduce `Exponential Backoff And Jitter` here when retries?

Reply:
We can consider using `tokio_retry::Retry`

Reply:
Good point!

Reply:
How to abort using https://github.com/Xuanwo/backon. Seems tokio_retry is not update for a long time.

Reply:
backon LGTM